IFC4x3/Properties/s/SegmentType_3JQtIbIuv9iuE9bxPSMuVX/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="SegmentType" UniqueId="d36b74a5-4b8e-49b3-8389-97b65c5b87e1" />
```

Comment:
No PrimaryDataType set.

Reply:
This file should be deleted. We decided to use IfcArchElementType/SEGMENT instead.

Reply:
> This file should be deleted. We decided to use IfcArchElementType/SEGMENT instead.

Deleted in https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/856ac09196cf19c2f696cc353c217a840768c34b